code/addons/test/src/node/vitest-manager.ts

```diff
@@ -1,4 +1,5 @@
 import { existsSync } from 'node:fs';
+import { dirname, join } from 'node:path';
```

Comment:
We've agreed in the team to use `pathe` instead of `node:path` when we can.

(should have an eslint rule I guess. 🤷)

Reply:
I did not find other issues, but I figured I'd be consistent and figured it would fail sooner or later.